Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -264,13 +276,15 @@ be written as:
     File; EXT:{extkey}/Configuration/Services.yaml
 ..  _EventDispatcherRegistration:
 
-Registering the event listener via :file:`Services.yaml`
---------------------------------------------------------
+Registering the event listener via :file:`Services.yaml` (legacy)
+-----------------------------------------------------------------
 
 ..  versionadded:: 13.0
     If using the PHP attribute :php:`\TYPO3\CMS\Core\Attribute\AsEventListener`
     to configure an event listener, the registration in the
     :file:`Configuration/Services.yaml` file is not necessary anymore.
+    However, extensions also supporting TYPO3 v12 may still
```

Comment:
Again, this makes backporting harder. Please move this in a separate PR only applied to main afterwards.

Reply:
Agreed, I see. Might not even be needed as a separate PR, it's a non-information, when reading again.